_posts/2019-12-22-hive-3.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title:  "Hive 3 support in Presto"
+author: Piotr Findeisen, Starburst Data
+excerpt_separator: <!--more-->
+---
+
+Hive community is centered around a few different Hive distributions, one of them
+being Hortonworks Data Platform (HDP). Even after Cloudera-Hortonworks merger there
+is vivid interest in HDP 3, featuring Hive 3. Presto is ready for the game.  
+
+<!--more-->
+
+# Introduction
+
+There are several Hive versions being in active use by the Hive community: 0.x, 1.x, 2.x
+and 3.x. Hive 3 major release brings a number of interesting features, including:
+
+* support for Hadoop Erasure Coding (EC), allowing [much better HDFS storage capacity
```

Comment:
I'd remove all the commas at the end of the items, since this is a list and not a sentence

Reply:
"Hive 3 brings features, including: ...." -- to me it's a sentence.
Anyway, do you want me to replace comas with periods? Or nothing at all, also on the final one?